CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] Use github.com/grafana/dskit/backoff instead of pkg/util/backoff.go.
```

Comment:
I don't think we need changelog entry for this, as this isn't user-visible change.

Reply:
@pstibrany OK, I will remove it :)